crossplane/02-eks/0-eks-iam-role.yml

```
@@ -0,0 +1,30 @@
+          }
+        ]
+      }
+---
+apiVersion: iam.aws.upbound.io/v1beta1
+kind: RolePolicyAttachment
+metadata:
+  name: easyshop-eks-cluster-role-amazon-eks-cluster-policy
+spec:
+  forProvider:
+    role: easyshop-eks-cluster-role
+    policyArn: arn:aws:iam::aws:policy/AmazonEKSClusterPolicy
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)